src/basic/ml/FStar_Util.ml

```diff
@@ -75,7 +75,8 @@ type proc =
      outc : out_channel;
      mutable killed : bool;
      stop_marker: (string -> bool) option;
-     id : string}
+     id : string;
+     start_time : time}
```

Comment:
Keeping this field for some profiling and testing, although it isn't really used in this PR